scripts/release_support/image-based-commit-tag.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
Can we update https://github.com/Seagate/cortx-re/blob/main/scripts/release_support/git-tag.sh instead of the new script?

Reply:
Thought to keep it for backup as this script do tagging based on RELEASE.INFO and new script does based on image.

Reply:
We can keep it as a backup in the branch instead of an additional script.

Reply:
took backup in branch - https://github.com/Seagate/cortx-re/tree/releaseinfo-commit-tag